frontend/lib/src/components/widgets/DateInput/DateInput.tsx

```diff
@@ -381,6 +381,7 @@ class DateInput extends React.PureComponent<Props, State> {
                       borderRightWidth: theme.sizes.borderWidth,
                       borderTopWidth: theme.sizes.borderWidth,
                       borderBottomWidth: theme.sizes.borderWidth,
+                      // paddingRight: 0,
```

Comment:
nit: can this be removed?

Reply:
oh yeah that's a left over from the other solution where I wanted to use margin-right with a positive value